jellyfish/2.2.3/Dockerfile

```diff
@@ -0,0 +1,54 @@
+#################################################################
+# Dockerfile
+#
+# Version:          1
+# Software:         jellyfish
+# Software Version: 2.2.3
+# Description:      A fast multi-threaded k-mer counter
+# Website:          https://github.com/gmarcais/Jellyfish
+# Tags:             Genomics
+# Provides:         jellyfish 2.2.3
+# Base Image:       biodckr/biodocker
+# Build Cmd:        docker build --rm -t biodckrdev/jellyfish 2.2.3/.
+# Pull Cmd:         docker pull biodckrdev/jellyfish
+# Run Cmd:          docker run --rm -it biodckrdev/jellyfish <options> <files>
+#################################################################
+
+# Set the base image to Ubuntu
+FROM biodckr/biodocker
```

Comment:
same remarks, should use biocontainers base image and use LABEL for metadata (+ license info)